clients/src/test/java/org/apache/kafka/common/utils/MockTime.java

```diff
@@ -20,13 +20,20 @@
 public class MockTime implements Time {
 
     private long nanos = 0;
+    private long autoTickMs = 0;
 
     public MockTime() {
         this.nanos = System.nanoTime();
     }
 
+    public MockTime(long autoTickMs) {
+        this.nanos = System.nanoTime();
+        this.autoTickMs = autoTickMs;
```

Comment:
Shouldn't this also autoTick `nanoseconds`? It seems like it could be confusing if someone changed some code from using `milliseconds` to `nanoseconds` and suddenly the test would start failing.